Sources/MapboxMaps/Style/StyleSourceManager.swift

```diff
@@ -168,11 +161,7 @@ internal final class StyleSourceManager: StyleSourceManagerProtocol {
         guard let data = data else { return }
         if case GeoJSONSourceData.empty = data { return }
 
-        if commonSettings.shouldUseDirectGeoJSONUpdate {
-            directlyApplyGeoJSON(data: data, sourceId: id)
-        } else {
-            applyGeoJSONData(data: data, sourceId: id)
-        }
+        directlyApplyGeoJSON(data: data, sourceId: id)
     }
 
     private func directlyApplyGeoJSON(data: GeoJSONSourceData, sourceId id: String) {
```

Comment:
Maybe the name can be changed now as there's just one method now
```suggestion
    private func applyGeoJSON(data: GeoJSONSourceData, sourceId id: String) {
```